apps/precinct-scanner/src/screens/PollWorkerScreen.tsx

```diff
@@ -83,6 +84,17 @@ const PollWorkerScreen = ({
     void calculateTally()
   }, [election, getCVRsFromExport, scannedBallotCount])
 
+  useEffect(() => {
+    void (async () => {
+      if (window.kiosk && window.kiosk.totp) {
+        const totpResult = await window.kiosk.totp.get()
```

Comment:
Could do `await window.kiosk?.totp?.get()` instead of the surrounding conditional. Though why would `.totp` not be set? Just trying to allow for the older version?

Reply:
yeah, don't want to start throwing errors on older versions just yet.